### PR TITLE
Overmap cabin sprites

### PR DESCRIPTION
**Pull request by acepleiades**
#### Summary
Ultica "Overmap cabin sprites"

#### Content of the change
Adds overmap cabin sprites

### CODE REVIEW

**Comment by acepleiades** [comment]
cabins
I think it works but, I haven't found other cabins to see if they rotate and I can't seem to spawn them correctly

File: gfx/UltimateCataclysm/pngs_normal_32x32/overmap/cabin/cabin.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      "cabin_5_north",
+      "cabin_6_north",
+      "cabin_7_north",
+      "cabin_lapin_north",
+      "bandit_cabin",
+      "cabin_isherwood_north",
+      "cabin_lake_north",
+      "lake_cabin_boathouse_north",
+	  "cabin_strange_north",
+	  "cabin_strange_b_north"
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_normal_32x32/overmap/cabin/cabin.json:18]:
```suggestion
	    "cabin_strange_north",
	    "cabin_strange_b_north"
```

File: gfx/UltimateCataclysm/pngs_normal_32x32/overmap/cabin/cabin.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      "cabin_5_east",
+      "cabin_6_east",
+      "cabin_7_east",
+      "cabin_lapin_east",
+      "bandit_east",
+      "cabin_isherwood_east",
+      "cabin_lake_east",
+      "lake_cabin_boathouse_east",
+	  "cabin_strange_east",
+	  "cabin_strange_b_east"
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_normal_32x32/overmap/cabin/cabin.json:39]:
```suggestion
	    "cabin_strange_east",
	    "cabin_strange_b_east"
```

File: gfx/UltimateCataclysm/pngs_normal_32x32/overmap/cabin/cabin.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      "cabin_5_south",
+      "cabin_6_south",
+      "cabin_7_south",
+      "cabin_lapin_south",
+      "bandit_south",
+      "cabin_isherwood_south",
+      "cabin_lake_south",
+      "lake_cabin_boathouse_south",
+	  "cabin_strange_south",
+	  "cabin_strange_b_south"
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_normal_32x32/overmap/cabin/cabin.json:60]:
```suggestion
	    "cabin_strange_south",
	    "cabin_strange_b_south"
```

File: gfx/UltimateCataclysm/pngs_normal_32x32/overmap/cabin/cabin.json [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      "cabin_5_west",
+      "cabin_6_west",
+      "cabin_7_west",
+      "cabin_lapin_west",
+      "bandit_west",
+      "cabin_isherwood_west",
+      "cabin_lake_west",
+      "lake_cabin_boathouse_west",
+	  "cabin_strange_west",
+	  "cabin_strange_b_west"
```
Comment (int-ua) [inline gfx/UltimateCataclysm/pngs_normal_32x32/overmap/cabin/cabin.json:81]:
```suggestion
  	  "cabin_strange_west",
  	  "cabin_strange_b_west"
```

**Comment by acepleiades** [comment]
I think the rotations are not displaying correctly just like https://github.com/I-am-Erk/CDDA-Tilesets/pull/705

**Comment by acepleiades** [comment]
cabin rotate
I got the cabin to rotate correctly

**Pull request merged**